Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: eject-packages
+eject-packages:
+	rm -rf mech_client/helpers/p2p_libp2p_client
+	cp -r packages/valory/connections/p2p_libp2p_client mech_client/helpers/p2p_libp2p_client
+
+	rm -rf mech_client/helpers/acn
+	cp -r packages/valory/protocols/acn mech_client/helpers/acn
+
+	rm -rf mech_client/helpers/mech_acn
+	cp -r packages/valory/protocols/mech_acn mech_client/helpers/mech_acn
+
+.PHONY: copy-packages-local
+copy-packages-local:
```

Comment:
Temporary until the `mech_acn` protocol gets published, after that we'll use sync